x/cert/keeper/migrations.go

```diff
@@ -0,0 +1,22 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	v280 "github.com/shentufoundation/shentu/v2/x/cert/legacy/v280"
+)
+
+// Migrator is a struct for handling in-place store migrations.
+type Migrator struct {
+	keeper Keeper
+}
+
+// NewMigrator returns a new Migrator.
+func NewMigrator(keeper Keeper) Migrator {
+	return Migrator{keeper: keeper}
+}
+
+// Migrate1to2 migrates from version 1 to 2.
+func (m Migrator) Migrate1to2(ctx sdk.Context) error {
```

Comment:
There is a `libraryStoreKey` in cert store, which contains string formatted address. But its entry only appears in genesis, IDK if our mainnet has any

Reply:
At x/cert/genesis.go:L44 and x/cert/keeper/library.go:L10